src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -838,11 +838,22 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
 
     private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container)
     {
-        $container
+        $definition = $container
             ->getDefinition('property_accessor')
             ->replaceArgument(0, $config['magic_call'])
             ->replaceArgument(1, $config['throw_exception_on_invalid_index'])
         ;
+
+        if (isset($config['cache']) && $config['cache']) {
+            $container->setParameter(
+                'property_accessor.cache.prefix',
```

Comment:
This code has been copied from the validator and it is used in some other part.

IIRC, the goal is not especially to handle the deployment logic but to allow several SF apps to be hosted on a shared server without conflicts. (If 2 apps have a class called `AppBundle\Entity\User`, it will be no conflict thanks to this line).